CredScanSuppressions.json

```diff
@@ -11,6 +11,7 @@
     },
     {
       "file": [
+        "src\\azure-cli\\azure\\cli\\command_modules\\appconfig\\tests\\latest\\recordings\\test_appconfig_to_appservice_import_export.yaml",
```

Comment:
Could you tell which secret/password is against the CredScan?

Reply:
my bad, cred scan suppression is not needed for this test because its a livescenario test. I'll remove it.